### PR TITLE
feature/storybook

### LINKED ISSUE #30: Add storybook to the project
As the name states, add storybook for isolated component development

### PR DESCRIPTION
**Pull request by abdigali1**
closes #30 
Added Storybook to our project, [off. docs](https://storybook.js.org/docs/guides/guide-react/) using react preset for storybook and typescript, scss configs to main.js

Changed tsconfig.json "noEmit" config, but with Storybook it has to be false. 
When To Use: When you want to use the TypeScript compiler for type checking, not to do compile. Normally Babel can handle converting the TypeScript file. In our project we have babel for type-checking, so we do not need "noEmit": true
Usefull source to understand tsconfig configs 
1) https://medium.com/extra-credit-by-guild/tsconfig-json-demystified-d8f333e578c1
2) https://medium.com/javascript-in-plain-english/typescript-configuration-options-tsconfig-json-561d4a2ad4b

### CODE REVIEW

File: .idea/vcs.xml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (aidarnouman) [inline .idea/vcs.xml]: bro, you accidentally pushed all of the `.idea` folder with various IDE config files - you need to add them to `.gitignore` and [delete them from git cache](https://stackoverflow.com/questions/13541615/how-to-remove-files-that-are-listed-in-the-gitignore-but-still-on-the-repositor#:~:text=gitignore%20are%20not%20being%20tracked,unix)%20will%20give%20you%20help.)

File: package.json [hunk 2]
```diff
@@ -111,6 +114,7 @@
     "node-fetch": "^2.6.0",
     "node-loader": "^0.6.0",
     "node-sass": "^4.14.1",
+    "npm": "^6.14.6",
```
Comment (aidarnouman) [inline package.json:117]: what this one is for?

File: webpack.renderer.config.js [hunk 3]
```diff
@@ -36,9 +36,10 @@ module.exports = {
       "@models": srcPaths("app/models"),
       "@renderer": srcPaths("app/renderer"),
```
Comment (aidarnouman) [inline webpack.renderer.config.js:37]: - why this one has path `"app/renderer"` and `@components` alias has path `./app/renderer/components` - I mean one of them has preceding dot while other isn't, are both of them working?
- can you please add some usage examples of aliases to our existing sample components (e.g. Heading), something like
```js
import { Heading } from "@components"
```

**Comment by nugmanoff** [comment]
Good reasoning & good job bro! Thanks for the context & links regarding `noEmit` thing👍

File: webpack.renderer.config.js (hunk 3 above)
Comment (abdigali1) [reply webpack.renderer.config.js:37]: When I decided which one to add, I test all of them srcPath("app"), path.resolve("app"), path.resolve(_dirname, "app") and all of them is working (I added some scss file to another directory, created webpack alies and added used that alias), but it is not workings in some places. Maybe you should add file format extension, as i added below aliases, but it works.

1) "@models": srcPaths("app/models"), "@renderer": srcPaths("app/renderer") this two was in the articles about boilerplate, I thought it is necessery) 
2) import { Heading } '@components/Heading'

**Comment by abdigali1** [comment]
After several attemps to add an webpack aliases, I researched and understood that, Typescript argued with this alias, because it doesn't know about this alias and tsconfig has own aliases:
`"paths": {
      "@static": ["./static"],
      "@styles": ["./static/styles"],
      "@components": ["./app/renderer/components"]
 }`
If you just write it in tsconfig aliases, it will also an error, because there is nothing to handle it and there you can use webpack.

I wrote usage example in **App.tsx**.
If you want to add more aliases, you should add it firts in webpack alias and then in tsconfig, like I did.

Useful resources: 
1. https://medium.com/@joshuaavalon/webpack-alias-in-typescript-declarations-81d2b6c0dcd6
2. https://www.basefactor.com/configuring-aliases-in-webpack-vs-code-typescript-jest

**Review by nugmanoff — APPROVED** [review]

**Pull request merged**